lib/internal/streams/destroy.js

```diff
@@ -8,7 +8,10 @@ function destroy(err, cb) {
     this._writableState.destroyed;
```

Comment:
this was done in the previous PR, I would prefer to discuss that in another issue, if you want to fire a PR (I would like to get this in ASAP, keeping it only on the regression).